exporter/exporter.go

```diff
@@ -316,6 +317,45 @@ type topEntry struct {
 	Value float64
 }
 
+func getFieldsAndTags(topic string, entry map[string]interface{}, tags map[string]string, fields map[string]string) (tagsMap map[string]string, fieldsMap map[string]interface{}) {
+	tagsMap = make(map[string]string)
+	for tag, value := range tags {
+		if len(value) > 1 && strings.HasPrefix(value, "#") {
```

Comment:
I'd prefer a more conventional form of variable representation here. Perhaps `$fieldName` or `${fieldName}`? It's easy for the # to be mistaken for a yaml comment.